loki-config.yaml

```
@@ -41,4 +41,4 @@ limits_config:
 
 compactor:
   working_directory: /etc/loki/compactor
-  shared_store: filesystem
+  shared_store: filesystem
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)